<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>知微客户端</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 设置按钮样式 */
#settings_btn {
    background-color: transparent;
    border: none;
    padding: 2px;
}
#settings_btn:hover {
    background-color: #e0e0e0;
    border-radius: 15px;
}

/* 下载按钮样式 */
QPushButton[objectName^="download_btn"] {
    background-color: transparent;
    border: none;
    padding: 2px;
}
QPushButton[objectName^="download_btn"]:hover {
    background-color: #e0e0e0;
    border-radius: 4px;
}

/* 删除按钮样式 */
QPushButton[objectName^="delete_btn"] {
    background-color: transparent;
    border: none;
    padding: 2px;
}
QPushButton[objectName^="delete_btn"]:hover {
    background-color: #e0e0e0;
    border-radius: 4px;
}

/* 查看详情按钮样式 */
QPushButton[objectName^="view_btn"] {
    background-color: transparent;
    border: none;
    padding: 2px;
}
QPushButton[objectName^="view_btn"]:hover {
    background-color: #e0e0e0;
    border-radius: 4px;
}

/* 状态栏按钮样式 */
QPushButton[objectName^="status_btn"] {
    padding: 2px;
    border-radius: 4px;
}
QPushButton[objectName^="status_btn"]:hover {
    background-color: #e0e0e0;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="main_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="main_splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="navigation_panel">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="navigation_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="search_widget">
          <layout class="QHBoxLayout" name="search_layout">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLineEdit" name="search_edit">
             <property name="placeholderText">
              <string>请输入要搜索的用户</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="task_title">
          <property name="styleSheet">
           <string notr="true">background-color: #f0f0f0; padding: 10px; color: black; font-weight: bold;</string>
          </property>
          <property name="text">
           <string>云端测区任务</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomTreeWidget" name="test_tree"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="content_area">
       <property name="styleSheet">
        <string notr="true">background-color: #e6e9f0;</string>
       </property>
       <layout class="QVBoxLayout" name="content_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWebEngineView" name="web_view"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>custom_widgets</header>
  </customwidget>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>PySide6.QtWebEngineWidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui> 